app/packages/core/src/plugins/SchemaIO/components/NativeModelEvaluationView/Evaluation.tsx

```
@@ -0,0 +1,1303 @@
+                                    <GridView sx={{ fontSize: 16 }} />
+                                  </IconButton>
+                                )}
+                              </Stack>
+                            </TableCell>
+                            <TableCell>
+                              <Stack
+                                direction="row"
+                                sx={{ justifyContent: "space-between" }}
+                              >
+                                <Typography>{difference}</Typography>
+                                {!isNaN(ratio) && (
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)